modules/integration/tests-ui-integration/tests-ui/src/test/java/org/wso2/ds/ui/integration/test/dashboard/GadgetUsageTest.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.ds.ui.integration.test.dashboard;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.*;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.integration.common.utils.exceptions.AutomationUtilException;
+import org.wso2.ds.ui.integration.util.DSUIIntegrationTest;
+
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+
+import static org.testng.Assert.*;
+
+/**
+ * Tests related with the gadget usage related warnings
+ */
+public class GadgetUsageTest extends DSUIIntegrationTest {
+    private static final String DASHBOARD1_TITLE = "gadgetusagedashboard1";
+    private static final String DASHBOARD2_TITLE = "gadgetusagedashboard2";
+
+    /**
+     * Initializes the class.
+     *
+     * @param userMode user mode
+     */
+    @Factory(dataProvider = "userMode")
+    public GadgetUsageTest(TestUserMode userMode) {
+        super(userMode);
+    }
+
+    /**
+     * Provides user modes.
+     *
+     * @return user modes
+     */
+    @DataProvider(name = "userMode")
+    public static Object[][] userModeProvider() {
+        return new Object[][] { { TestUserMode.SUPER_TENANT_ADMIN } };
```

Comment:
No need to leave these unnecessary spaces when declaring a array. Please use a code formatting tool which will help in this case.